x/harvest/types/params.go

```diff
@@ -218,26 +221,97 @@ func (ds DistributionSchedule) GetMultiplier(name MultiplierName) (Multiplier, b
 // Multipliers slice of Multiplier
 type Multipliers []Multiplier
 
+// BorrowLimit enforces restrictions on a money market
+type BorrowLimit struct {
+	MaximumLimit sdk.Int `json:"maximum_limit" yaml:"maximum_limit"`
+	LoanToValue  sdk.Dec `json:"loan_to_value" yaml:"loan_to_value"`
+}
+
+// NewBorrowLimit returns a new BorrowLimit
+func NewBorrowLimit(maximumLimit sdk.Int, loanToValue sdk.Dec) BorrowLimit {
+	return BorrowLimit{
+		MaximumLimit: maximumLimit,
+		LoanToValue:  loanToValue,
+	}
+}
+
+// Validate BorrowLimit
+func (bl BorrowLimit) Validate() error {
+	if bl.MaximumLimit.IsNegative() {
+		return fmt.Errorf("maximum limit cannot be negative: %s", bl.MaximumLimit)
+	}
+	if !bl.LoanToValue.IsPositive() {
+		return fmt.Errorf("loan-to-value must be a positive integer: %s", bl.LoanToValue)
+	}
+	if bl.LoanToValue.GT(sdk.NewDec(100)) {
+		return fmt.Errorf("loan-to-value cannot be greater than 100: %s", bl.LoanToValue)
+	}
+	return nil
+}
+
+// MoneyMarket is a money market for an individual asset
+type MoneyMarket struct {
+	Denom        string      `json:"denom" yaml:"denom"`
+	BorrowLimit  BorrowLimit `json:"borrow_limit" yaml:"borrow_limit"`
+	SpotMarketID string      `json:"spot_market_id" yaml:"spot_market_id"`
```

Comment:
Add `ConversionFactor`

Reply:
Included in https://github.com/Kava-Labs/kava/pull/702/commits/277c4b1936981182fc09ae0c6ea0973d9cbbca8a